packaging/suse/amazon-ecs-init.spec

```diff
@@ -0,0 +1,91 @@
+#
+# spec file for package amazon-ecs-init
+#
+# Copyright (c) 2015 SUSE LINUX Products GmbH, Nuernberg, Germany.
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via http://bugs.opensuse.org/
+#
+
+%define short_name amazon-ecs
+Name:           amazon-ecs-init
+Version:        1.4.0.2
```

Comment:
If we do not encode the version somehow, and -x is not allowed in the version how would we differentiate 1.4.0-1 and 1.4.0-2 if the spec file remains at Version: 1.4.0

If I cannot make the differentiation between 1.4.0-1 and 1.4.0-2 in the spec file version declaration we will be stuck with always the first release. I don't think this is in our best interest. Adding the .2 at the end was the most direct way I could think of to capture the -X naming convention used for the tarball.